client/src/pages/learn/responsive-web-design/learn-css-variables-by-building-a-city-skyline/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Learn CSS Variables by Building a City Skyline
+block: Learn CSS Variables by Building a City Skyline
```

Comment:
I am not sure if this was your doing, or if some of them just look like this 🤷‍♂️ 

```suggestion
block: learn-css-variables-by-building-a-city-skyline
```